pkg/apiutil/client.go

```diff
@@ -68,3 +69,20 @@ func (c *Client) GetLeader() (*pdpb.Leader, error) {
 	}
 	return leader, nil
 }
+
+// GetClusterID returns the Cluster ID.
+func (c *Client) GetClusterID() (uint64, error) {
+	clusterURL := c.url + "/cluster"
```

Comment:
`/cluster` needs we bootstrap tikv first, it is not convenient when run `pd-tso-bench`   we can use `/members` instead